templates/zerver/help/move-content-to-another-topic.md

```diff
@@ -22,8 +22,8 @@ for the details on when topic editing is allowed.
 
 {!message-actions-menu.md!}
 
-1. Select the first option. It may be called **View source / Move message**,
-   or simply **Edit message**. If it's called **View source**, then you are not
+1. Select the first option. It may be called **Move message**,
+   or simply **Edit message**. If it's called **View message source**, then you are not
    allowed to edit the topic of that message.
```

Comment:
These should be just "Select 'Move message'" now, right?

Reply:
It would be "Edit message" when editing content is also allowed. And "Move message" when only topic edit is allowed.

Reply:
Yeah, we don't currently show "Move message" when editing is allowed. Do you think we should?

Reply:
For the actual text of the instructions, let's change this line to something like:

> Select "Move message" or "Edit message" (only one of these options will be shown). If you do not see either option, you do not have permission to move this message.

Reply:
> Yeah, same as in `main`, we don't currently show "Move message" when editing is allowed. Do you think we should? That might be even cleaner...

Let's discuss this part on CZO: https://chat.zulip.org/#narrow/stream/101-design/topic/three-dot.20message.20menu/near/1441015